index.bs

```diff
@@ -115,29 +119,36 @@ Long Task timing involves the following new interfaces:
     };
 </pre>
 
-{{PerformanceLongTaskTiming}} extends the following attributes of {{PerformanceEntry}} interface:
+The values of the attributes of a {{PerformanceLongTaskTiming}} are set in the processing model in [[#report-long-tasks]]. The following provides an informative summary of how they will be set.
+
+The {{PerformanceEntry/name}} attribute's getter will return one of the following strings:
 
-* The {{PerformanceEntry/name}} attribute must return a {{DOMString}} that provides minimal frame attribution.
+: "<code><dfn>unknown</dfn></code>"
```

Comment:
Is it possible to indent "uknown" here so that it's to the right of the sentence "The name attribute's getter[...]"?

Reply:
If we switch to using a `<dl>` explicitly we could put a class on it. But this is pretty standard styling for definition lists, I think... IMO the colon makes it clear enough.

Reply:
Ah ok if it's standard then lgtm! I was just mentioning this from the HTML view not from the bikeshed file: currently "unknown" is same indentation as the previous sentence, and to me it would be slightly better if it was more indented than the previous sentence. No strong opinions though.